conda_build/noarch_python.py

```diff
@@ -131,7 +151,7 @@ def transform(m, files):
         link_code = fi.read()
 
     # Write the package metadata, and bumper with code for linking
-    with open(join(prefix, 'link.py'), 'w') as fo:
+    with open(join(prefix, 'link.py'), 'w', newline='\n') as fo:
```

Comment:
This case is not strictly necessary, but I reckoned it would be cleaner to be consistent. I'd be happy to get rid of all the newline stuff, though (except for the .bat and .sh).